docs/migration/4_0.md

```diff
@@ -1,3 +1,48 @@
 # Technical Changes in Spartacus 4.0
 
 ## Breaking Changes Introduced in 4.0
+
+### UserGroupUserListComponent
+- Removed `MessageService` from constructor.
+
+### ToggleStatusComponent
+- Removed `FeatureConfigService` from constructor.
+- Added `DisableInfoService` to constructor.
+
+### DeleteItemComponent
+- Removed `FeatureConfigService` from constructor.
+
+### UnitChildrenComponent
+- Added `CurrentUnitService` to constructor.
+
+### UnitCostCenterListComponent
+- Added `CurrentUnitService` to constructor.
+
+### UnitUserListComponent
+- Added `CurrentUnitService` to constructor.
+
+### UnitFormComponent
+- Renamed property `formGroup` to `form`.
+- Removed property `form$`.
```

Comment:
Can you add automatic migration for these changes with `methods-and-properties-deprecations` mechanism?

Reply:
done